recipe/meta.yaml

```diff
@@ -13,6 +13,8 @@ build:
   number: 0
   run_exports:
     - {{ pin_subpackage('minizip', max_pin='x') }}
+  ignore_run_exports:
+    - gtest
 
 requirements:
   build:
```

Comment:
could probably run with ninja on unix too, probably only worth making the change if the build currently takes a long time though

Reply:
This seems pretty quick.